examples/https/client.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const config = require('./setup');
+/**
+ * The trace instance needs to be initialized first, if you want to enable
+ * automatic tracing for built-in plugins (HTTPs in this case).
+ */
+config.setupTracerAndExporters('https-client-service');
+
+const https = require('https');
+const tracer = opentelemetry.getTracer();
+
+/** A function which makes requests and handles response. */
+function makeRequest() {
+    // span corresponds to outgoing requests. Here, we have manually created
+    // the span, which is created to track work that happens outside of the
+    // request lifecycle entirely.
+    const span = tracer.startSpan('makeRequest');
+    tracer.withSpan(span, () => {
+        https.get({
+            host: 'localhost',
+            port: 443,
+            path: '/helloworld',
+            agent: new https.Agent({  
```

Comment:
Just curious to know, why do we need this? What would happen if we skip this?

Reply:
it still works because in setup I have `process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';` and it doesn't work if I remove `process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';`